lib/payment/api_call.rb

```diff
@@ -32,10 +32,11 @@ def get_payment(account_id, payment_id, cents = false)
 
   # Create new payment
   #
+  # @param account_id [String] ID of the account related to Payment object to be created.
   # @param payment [Payment] payment object to be created. It should not have a payment_id set, Required
   # @return [Payment] newly created `Payment` object
-  def create_payment(payment)
-    query_api_object Model::Payment, "/rest/accounts/#{payment.account_id}/payments", payment.to_hash, 'POST'
+  def create_payment(account_id, data)
+    query_api_object Model::Payment, "/rest/accounts/#{account_id}/payments", data, 'POST'
```

Comment:
Sometimes we use parentheses (https://github.com/netguru/ruby-figo/pull/14/files#diff-e1f0d126df2bbae2ca53c780ca218f8bR58) and sometimes they are omitted ☝️. I suggest picking one way for the sake of consistency 😄